inc/dropdown.class.php

```diff
@@ -2247,24 +2255,25 @@ public static function getDropdownValue($post, $json = true) {
       }
 
       if (isset($post['condition']) && !empty($post['condition'])) {
-         if (isset($_SESSION['glpicondition'][$post['condition']])) {
-            $post['condition'] = $_SESSION['glpicondition'][$post['condition']];
+         $key = sha1(serialize($post['condition']));
```

Comment:
Maybe I am wrong, but I think `$post['condition']` contains the key (and is defined here https://github.com/glpi-project/glpi/pull/5006/files?utf8=%E2%9C%93#diff-527ac4764462540ccf1068f5fa312137R1437 ).

Reply:
Condition parameter sent to the ajax call is the SHA1SUM returned from addNewCondition method.